### DISCUSSION TITLE
Pockethost doesn't send response when using auth-with-password

### DISCUSSION THREAD

**Discussion by argonptg** [pocketbase/pocketbase#4242] [Q&A] [answered]
Hello! Me and @Y0URD34TH have been adding a implementation for a community-ish thing, and it worked fine for a while, but after some time it just stopped sending the response, leaving some users unable to log-in

Here are some pictures of DevTools (inside the app running CEF)

![image](https://github.com/pocketbase/pocketbase/assets/94653435/b368ab59-fc31-49bc-a3f1-1807663093f1)
![image](https://github.com/pocketbase/pocketbase/assets/94653435/65914c90-5f49-46f5-a455-6d2f2fcad6fd)
![image](https://github.com/pocketbase/pocketbase/assets/94653435/ce0d1435-23f0-40ac-9025-ee814fe9d4f5)

And here are some screenshots of Firefox

![image](https://github.com/pocketbase/pocketbase/assets/94653435/8ead4520-41ac-4b0d-a216-971d0bd794fd)
![image](https://github.com/pocketbase/pocketbase/assets/94653435/52b0728c-bb13-4b6b-8b28-f043a082011b)
![image](https://github.com/pocketbase/pocketbase/assets/94653435/7c4b6cf1-2498-4b6c-86a6-a9ed7b37e461)

I am using pocketbase v0.20.7 and Pockethost (I'll be also contacting them)

**Comment by ganigeorgiev** [comment, marked as answer]
I don't know what is the cause of the issue but I don't think this is PocketBase related.

If you are able to reproduce it locally, feel free to provide some steps and I'll try to investigate it.

**Reply by argonptg** [reply]
Hey after we tried running on firefox (same machine as @Y0URD34TH) to (at least) try reproducing the problem the response came different, this time it only came back with the record, not including the token

![image](https://github.com/pocketbase/pocketbase/assets/94653435/a720d7aa-d3c6-4de6-ac4f-e8b1a2406c1a)

so this is seeming more like a pockethost issue than a pocketbase issue, still, thank you for responding!
